src/x509.c

```diff
@@ -7945,23 +7966,23 @@ void wolfSSL_X509_CRL_free(WOLFSSL_X509_CRL *crl)
 #endif /* HAVE_CRL && (OPENSSL_EXTRA || WOLFSSL_WPAS_SMALL) */
 
 #ifdef OPENSSL_EXTRA
-#ifndef NO_WOLFSSL_STUB
 WOLFSSL_ASN1_TIME* wolfSSL_X509_CRL_get_lastUpdate(WOLFSSL_X509_CRL* crl)
 {
-    (void)crl;
-    WOLFSSL_STUB("X509_CRL_get_lastUpdate");
-    return 0;
+    if ((crl != NULL) && (crl->crlList->lastDate[0] != 0)) {
+        return (WOLFSSL_ASN1_TIME*)crl->crlList->lastDate;
```

Comment:
lastDate is a byte array. Don't think you can simply cast it to WOLFSSL_ASN1_TIME*. Likewise for wolfSSL_X509_CRL_get_nextUpdate and nextDate.

Reply:
I am changing struct CRL_Entry and struct DecodedCRL to use WOLFSSL_ASN1_TIME instead of just having byte    lastDate[MAX_DATE_SIZE] and byte    lastDateFormat. This will (is) going to change other code which uses this struct.

Reply:
CRL_Entry -- sounds good. That's an internal struct, so we are free to change its members. DecodedCRL -- that could be trickier. It's in asn.h, which we do export to users. @dgarske what do you think? I know asn_public.h is intended to be the "public ASN header," but asn.h is in fact public, too...